common/data_refinery_common/message_queue.py

```diff
@@ -85,9 +76,19 @@ def send_job(job_type: Enum, job, is_dispatch=False) -> bool:
     else:
         raise ValueError("Invalid job_type: {}".format(job_type.value))
 
-    logger.debug(
-        "Queuing %s nomad job to run job %s with id %d.", nomad_job, job_type.value, job.id
-    )
+
+def is_job_processor(job_type):
+    if job_type in list(Downloaders):
+        return False
+    elif job_type in list(SurveyJobTypes):
+        return False
+
+    return True
```

Comment:
```suggestion
def is_job_processor(job_type):
    return job_type not in list(Downloaders) and job_type not in list(SurveyJobTypes)
```

is this clearer to you?